Assets/Scripts/Gameplay/Action/ConcreteActions/ChargedShieldAction.cs

```diff
@@ -150,6 +150,8 @@ private void StopChargingUp(ServerCharacter parent)
 
```

Comment:
adding comment here to not forget contribution checklist, everything else looks good.